src/bromwhich_contour.jl

```diff
@@ -0,0 +1,138 @@
+### Implement Laplace transform along a hyperbola contour ###
+
+# 1.
+# Weideman, J., and L. Trefethen.
+# Parabolic and hyperbolic contours for computing the Bromwich integral.
+# Mathematics of Computation 76.259 (2007): 1341-1356.
+
+# 2.
+# Liemert, André, and Alwin Kienle.
+# Application of the Laplace transform in time-domain optical spectroscopy and imaging.
+# Journal of biomedical optics 20.11 (2015): 110502.
+
+### adaptive contour for each time point (can't precompute f(s) or sk)
+
+# parameterized hyperbola contour
+# parameters are given by ref [2] which are given in increased precision from calculations given in [1]
+function s(θ, N, t)
```

Comment:
`s` and `ds` should be combined in a single functions as they share several arithmetic operations. Can save at least one multiplication and division per loop doing this.

I don't think there's an option to return both `sinh` and `cosh` as there is a `sincos(x)` function.